logic.js

```diff
@@ -0,0 +1,116 @@
+const Tweeter = function () {
+
+    const _posts = []
+        
+    let postIdCounter = 0
+    let commentIdCounter = 0
+
+    const getPosts = function () {
+        return _posts
+    }
+
+    const addPost = function (text) {
+        if(text!="") {
+        postIdCounter++
+        const newPost = {
+            id: "p" + postIdCounter,
+            text: text,
+            comments: [] 
+        }
+        _posts.push(newPost)
+    }
+    }
+
+    const removePost = function (postID) {
+        let eleIndex = -1
+        for ( i in _posts) {
```

Comment:
When declaring a new variable (`i`) you should declare it with  _let_, otherwise, the variable will be defined as "_var_"